src/st/no3/377D4.c

```diff
@@ -475,12 +492,20 @@ Entity* AllocEntity(Entity* start, Entity* end) {
     return NULL;
 }
 
-INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801C5534);
+#ifndef NON_MATCHING
+INCLUDE_ASM("asm/st/no3/nonmatchings/377D4", func_801C5534); // not mathching only locally: https://decomp.me/scratch/6ZStN
```

Comment:
Please do not put comments at the same line of `INCLUDE_ASM` as `make format` will mess up the line and `make extract` will break. Put the comments above `INCLUDE_ASM`